mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -262,9 +263,10 @@ class SearchlightHyperalignment(ClassWithCollections):
             If None, hyperalignment is performed at every voxel (default).""")
 
     hyperalignment = Parameter(
-        Hyperalignment(ref_ds=0),
+        Hyperalignment(ref_ds=None),
         doc="""Hyperalignment instance to be used in each searchlight sphere.
             Default is just the Hyperalignment instance with default parameters.
+            Its `ref_ds` parameter would be overridden and therefore has no effect.
```

Comment:
What/how it will be overridden?  I.e. how its value decided -- please describe here, and if @swaroopgj blesses the changes as regarding the functionality, I will merge